Replace adviser promo in the apply section (#4409)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md

  - apply
  - how to apply
  - applying for teacher training
cta_adviser:
  adviser:

>     text: An adviser with years of teaching experience can help you understand why your application was unsuccessful and what you can do next. Chat by phone, text or email, as little or as often as you need.

Reviewer: I'm not sure they can help with understanding why the application was unsuccessful - it depends how much information the provider has supplied?

Could we make this a bit more positive re supporting you to reapply / maximise your chances of success?

>     text: An adviser with years of teaching experience can help you understand why your application was unsuccessful and what you can do next. Chat by phone, text or email, as little or as often as you need.

Author: ```suggestion
    text: An adviser with years of teaching experience can help you reapply for teacher training and maximise your chances of success. Chat by phone, text or email, as little or as often as you need.
```

>     text: Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat by phone, text or email, as little or as often as you need.

Reviewer: I think we could probably make this text a bit more relevant to the page here?

Author: ```suggestion
    text: An adviser with years of teaching experience can help you understand why your application was unsuccessful and what you can do next. Chat by phone, text or email, as little or as often as you need.
```
